include/gogoproto/gogo.proto

```diff
@@ -0,0 +1,136 @@
+// Protocol Buffers for Go with Gadgets
```

Comment:
Was this included on purpose for prost?

Reply:
Yeah, iirc Prost had errors without having this proto here. I'll update the README

Reply:
Hmm, I'm not clear on what the README is saying. It looks to me like it is just installing tools, not the protobuf specs, but I don't know the Go/protobuf situation.

Reply:
ping @Hoverbear do you think the README is saying something else?

Reply:
ping @Hoverbear PTAL at the above questions

Reply:
I'm so sorry I didn't see this nick. This email thread was buried behind a pagination. If I'm being so delinquent again please ping me is a more direct way so I can catch it. 🙇‍♀ 

Let's leave it as is.